test/cases/ostree-pulp.sh

```diff
@@ -0,0 +1,465 @@
+#!/bin/bash
+set -euo pipefail
+
+# Get OS data.
+source /usr/libexec/osbuild-composer-test/set-env-variables.sh
+source /usr/libexec/tests/osbuild-composer/shared_lib.sh
+
+# Get compose url if it's running on unsubscried RHEL
+if [[ ${ID} == "rhel" ]] && ! sudo subscription-manager status; then
+    source /usr/libexec/osbuild-composer-test/define-compose-url.sh
+fi
+
+# Provision the software under test.
+/usr/libexec/osbuild-composer-test/provision.sh none
+
+# Set os-variant and boot location used by virt-install.
+case "${ID}-${VERSION_ID}" in
+    "rhel-9.3")
+        IMAGE_TYPE=edge-commit
+        OSTREE_REF="rhel/9/${ARCH}/edge"
+        OS_VARIANT="rhel9-unknown"
+        # Use a stable installer image unless it's the nightly pipeline
+        BOOT_LOCATION="http://download.devel.redhat.com/released/rhel-9/RHEL-9/9.2.0/BaseOS/x86_64/os/"
```

Comment:
@yih-redhat Is this correct?  The distro version ID is 9.3 but we're using 9.2.0 to boot.

Reply:
Hi @achilleas-k This is copied from edge commit test case https://github.com/osbuild/osbuild-composer/blob/main/test/cases/ostree.sh#L111, and I also have the same question as you. I believe this is added by @henrywang , xiaofeng, could you please help to explain a bit more about this? 
Btw, this test script can pass without problem.

Reply:
It looks like it's just using the latest stable to avoid issues with the unreleased 9.3.  Resolving the conversation.